spec/classes/foreman_proxy__plugin__host_reports_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'foreman_proxy::plugin::host_reports' do
+  on_plugin_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+      let(:pre_condition) { 'include foreman_proxy' }
+
+      describe 'with default settings' do
+        it { should contain_foreman_proxy__plugin__module('host_reports') }
+        it 'host_reports.yml should contain the correct configuration' do
```

Comment:
I think this reads a bit more natural:
```suggestion
        it 'should contain the correct configuration in host_reports.yml' do
```